homeassistant/components/demo/air_quality.py

```diff
@@ -32,39 +32,27 @@ async def async_setup_entry(
 class DemoAirQuality(AirQualityEntity):
```

Comment:
We should probably remove this platform as we don't want people to get the wrong idea that they should build things based on it


https://developers.home-assistant.io/docs/core/entity/air-quality?_highlight=air&_highlight=qua